src/main/java/com/example/gradecalculator/entities/UserGrade.java

```diff
@@ -0,0 +1,36 @@
+package com.example.gradecalculator.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "user_grade")
+public class UserGrade {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_subject_id", nullable = false)
+    private UserSubject userSubject;
+
+
+    @ManyToOne
+    @JoinColumn(name = "grade_type_id", nullable = false)
+    private GradeType gradeType;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @Column(nullable = false)
+    private int grade;
+
+    public UserGrade() {
+    }
```

Comment:
you can omitt an empty constructor if it is the only one for the class

Reply:
yup